code/client/munkilib/appleupdates.py

```diff
@@ -640,7 +640,7 @@ def _GetPreferredLocalization(self, list_of_localizations):
         localizations. Returns a single language/localization name.'''
         # pylint: disable=no-self-use
         localization_preferences = (
-            munkicommon.pref('AppleSoftwareUpdateLanguages') or ['English'])
+            munkicommon.getAppleLanguages() or ['English'])
```

Comment:
And that's terribly non-deterministic. For the more common case of a single user of a machine, munkicommon.pref('AppleSoftwareUpdateLanguages') ensures a consistent language for Apple's update descriptions and therefore a better user experience.

I don't want to make the common case worse when attempting to improve the behavior for your less-common use case.